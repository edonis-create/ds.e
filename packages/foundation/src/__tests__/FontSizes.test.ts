import FontSize from '../FontSizes';

test('snapshots of font sizes',()=>{
expect(FontSize).toMatchSnapshot();
});