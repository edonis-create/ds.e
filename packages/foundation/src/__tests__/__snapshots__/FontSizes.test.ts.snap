// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots of font sizes 1`] = `
Object {
  "base": "base",
  "lg": "lg",
  "sm": "sm",
  "xl": "xl",
  "xs": "xs",
}
`;
